db/migrate/20220901020121_create_subscribers.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSubscribers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :subscribers, id: :uuid do |t|
```

Comment:
changes primary key to UUID to avoid exposing IDs on client network update calls to `/api/subcribers/:id`